python/ray/tests/test_object_manager.py

```diff
@@ -296,6 +296,83 @@ def driver():
     ray.get(driver.remote())
 
 
+@pytest.mark.timeout(30)
+def test_pull_bundles_admission_control(shutdown_only):
+    cluster = Cluster()
+    object_size = int(1e7)
+    num_objects = 10
+    num_tasks = 10
+    # Head node can fit all of the objects at once.
```

Comment:
Why is that? If we'd like to test the feature, aren't we supposed to make only 1 task to fit at a time at all nodes?

Reply:
We need to create all the objects somewhere, so we make the head node big. The worker node will try to fetch objects from the head node, but it can only fit one task at a time.

Reply:
Ah, I misunderstood the comment. It makes sense!